include/zephyr/devicetree.h

```diff
@@ -2221,6 +2229,16 @@
 #define DT_REG_ADDR_BY_NAME(node_id, name) \
 	DT_CAT4(node_id, _REG_NAME_, name, _VAL_ADDRESS)
 
+/**
```

Comment:
Same comments apply here and below as to the previous macro.

Reply:
Ok